docs/1_how_it_works/1_2_difference.md

```diff
@@ -0,0 +1,41 @@
+# Difference between Finetuner and Others
+
+There are several fancy machine learning libraries available,
+what makes Finetuner unique?
+
+## Finetuner care about the quality of Embeddings 🧬
+
+Finetuner is not designed to make better classification,
+sentiment analysis or object detection.
+Finetuner only care about the quality of embeddings.
+The fine-tuned model only produces embeddings.
```

Comment:
Finetuner only cares about the quality of embeddings, and this is what the fine-tuned model will produce.